info.mumble.Mumble.yml

```diff
@@ -1,6 +1,6 @@
 id: info.mumble.Mumble
-rename-desktop-file: mumble.desktop
 rename-icon: mumble
+rename-desktop-file: org.mumble_voip.mumble.desktop
```

Comment:
you probably need to add `rename-appdata-file: org.mumble_voip.mumble.appdata.xml` here as well. Which would solve the build failure.

Reply:
Thank you for the help